metricbeat/module/ceph/mgr_pool_disk/_meta/data.json

```diff
@@ -0,0 +1,31 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "ceph": {
+        "pool_disk": {
+            "id": 0,
+            "name": "rbd",
```

Comment:
I think it would make sense to put pool ids and names in common fields to help on correlation and filtering. We have them at least in `ceph.pool_disk.id/name` and in `ceph.mgr_osd_pool_stats.pool_id/name`. We could maybe create some common fields at the module level like `ceph.pool.id/name`.

If this makes sense I am ok with doing it in a future refactor, it may affect existing metricsets.

Reply:
(discussed offline)

Speaking about the domain language, OSD pool and disk pool are not synonyms.

Also introducing a `pool_disk` as a common field will affect existing the `pool_disk` metricset and implicate field data redundancy (pool ID and name stored twice).